source/docs/articles/sites/code/unsupported-modules-plugins.md

```diff
@@ -80,7 +80,12 @@ $baseUrl = '/ckfinder/userfiles/';
  **Solution**: The [documentation on Drupal.org](https://drupal.org/node/257026) for the module mentions the issues and the remedy, which is a cache clear operation. If you are unable to exclude cached data from your dumps or avoid migrating cache data, you should clear your site’s cache after importing the data.
 <hr>
 #### Plupload
-**Issue**: See [Using the tmp Directory](/docs/articles/sites/code/unsupported-modules-plugins/#using-the-tmp-directory) section below.
+**Issue**: This module uses the `tmp` directory which is not supported in a distributed environment. For more information, see [Using the tmp Directory](/docs/articles/sites/code/unsupported-modules-plugins/#using-the-tmp-directory) section below.
```

Comment:
What do mean by distributed environment? Maybe better to just keep Issue section the same and link to `Using the tmp directory` that explains it in more detail.